src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/SubmissionViewModel.java

```diff
@@ -104,6 +108,25 @@ public String getSubmissionCountText() {
     return submissionCountText.toString();
   }
 
+  /**
+   * Formats a descriptive string for the submission dialog about a submittable file. The string
+   * includes the file name and its last modification date.
```

Comment:
the doc says "... last modification data", but is it really a date or rather a dateTime?